.github/workflows/pr-visual-regression.yml

```diff
@@ -0,0 +1,33 @@
+name: PR Visual Regression
+on:
+  workflow_run:
+    workflows: [PR Build]
+    types:
+      - completed
+jobs:
+  pr-preview:
+    name: PR Chromatic
```

Comment:
These should match. I made the others match the file name.

Reply:
fixed

Reply:
The `pr-preview` key is still the same.

Reply:
sorry, should be fixed now